### ISSUE TITLE
Should old synonyms be offered in the redetermination list?

### ISSUE THREAD

**Issue by kitenetter** [BiologicalRecordsCentre/iRecord#1140]
I've just realised that if a verifier redetermines a record, they are able to use old synonyms that are not available for normal data entry. The example was for a micro-moth, where a verifier redetermined a record from A. treischkiella to A. petryi, but was confused because the record display then defaulted back to A. trietschkiella, which is the current recommended name in UKSI:

![image](https://user-images.githubusercontent.com/16039111/130297011-34c50f46-c0f3-42ea-a434-ab2993425b83.png)

Do we intend verifiers to be able to redetermine records to non-recommended names? It doesn't seem like a useful thing to do, was there a reason for it?

**Comment by johnvanbreda** [comment]
I thought that the "syn. of ..." display was clear enough, so the verifier could be trusted to use the correct name variant? In some cases I guess the verifier may have a different view on the taxonomically accepted name to the currently loaded version of UKSI and therefore will want to choose a synonym - though this reason is less likely to be relevant as the rate at which UKSI is updated improves.

It will be an easy change to remove synonyms from this list.

**Comment by kitenetter** [comment]
The verifier in this case hadn't understood the implications of "syn. of". But I think this one was a special case - we used to have A. petryi on the British list, then it got changed to A. treitschkiella, then it was shown that both were good species and the one we thought we had in Britain is actually the other one. This sort of name-swapping causes confusion all round, not helped by the UKSI not having caught up with the latest changes.

The ideal solution would be not to show synonyms by default, but to provide the verifiers with a tick-box to "show synonyms" just in case they need it. Can't imagine that it would be needed very often though, so it's not a priority.

My own preference would be **_not_** to show synonyms. If the change is easy, can we do that, bearing in mind that if other verifiers object we may have to undo it?
